docs/release-notes/8.3.asciidoc

```diff
@@ -5,6 +5,11 @@
 [[release-notes-8.3.3]]
 === 8.3.3
 
+[discrete]
+[[known-issue-8.3.3]]
+==== Known issues
+* An {endpoint-sec} bug on macOS and Linux can cause CPU spikes if Malware prevention is enabled on an {endpoint-cloud-sec} integration policy. When this happens, {endpoint-sec} is locked up and stops functioning normally. To avoid this issue, we recommend using {stack} 8.3.2 or earlier. If you are using 8.3.3 and have encountered this issue, you can temporarily resolve it by rebooting your computer and disabling Malware prevention on your {endpoint-cloud-sec} integration policy (https://github.com/elastic/endpoint/issues/22[#22]).
```

Comment:
> ...  Malware prevention ...

This will also happen if Malware is in detect mode. @ricardoungureanu can you confirm that?

> ... we recommend using {stack} 8.3.2 or earlier ...

I'm not sure what `{stack}` will render like, but users can use an 8.3.3 Kibana/Elasticsearch. It's Agent/Endpoint that need to be 8.3.2 or earlier for this workaround.

Reply:
Gotcha, I can change the attribute to {agent} (which renders as "Elastic Agent") if that's correct.

Reply:
Pointing to existing docs on Malware sounds good!

Reply:
Oh, and yeah, saying to use Elastic Agent 8.3.2 or earlier is correct.